### ISSUE TITLE
Alt Elements

### ISSUE THREAD

**Issue by cwclair** [cwclair/01-accessibility-review#5]
# Alt Elements

## User Story
AS A marketing agency
I WANT a codebase that follows accessibility standards
SO THAT our own site is optimized for search engines

## Acceptance Criteria
GIVEN a webpage meets accessibility standards
WHEN I view the icon and image elements
THEN I find accessible alt attributes

**Comment by cwclair** [comment]
I reconfigured the main image in order to apply alt text to it. Applied missing alt tags to the three secondary images and the three icons in the right sidebar.
